src/interfaces/stepper-interfaces/CreateRules/rules-doc-interface.ts

```diff
@@ -0,0 +1,5 @@
+export interface RulesDocIF {
+  name: string,
+  lastModified: number,
```

Comment:
A number should be fine here.  Its value is retrieve from the `File` object.  

>A number that represents the number of milliseconds since the Unix epoch.

https://developer.mozilla.org/en-US/docs/Web/API/File/lastModified

I did remove the commas from the interface however.

Reply:
Gotcha.

Since times and dates are overwhelmingly strings in JS, please add a line comment indicating what this is or where it comes from, so a future developer will know it's different from other times and dates. Thanks.

Reply:
added comments to interface